pydantic/dataclasses.py

```diff
@@ -107,7 +107,7 @@ def dataclass(
     kw_only: bool = False,
     slots: bool = False,
 ) -> Callable[[type[_T]], type[PydanticDataclass]] | type[PydanticDataclass]:
-    """A decorator used to create a Pydantic-enhanced dataclass, similar to standard Python `dataclasses`,
+    """A decorator used to create a Pydantic-enhanced dataclass, similar to the standard Python `dataclasses`,
```

Comment:
_(Sorry for the back and forth here!)_ I noticed `dataclasses` is plural, which would negative `the` – here's the correct grammar for each:
* **Plural:** similar to standard Python `dataclasses`, ...
* **Singular:** similar to the standard Python `dataclass`, ...